Tchap/target.yml

```diff
@@ -127,11 +128,18 @@ targets:
     - path: ../Riot/Modules/Common/Recents/Model/RecentCellData.m
     - path: ../Riot/Modules/Common/Recents/RecentsViewController.h
     - path: ../Riot/Modules/Common/Recents/RecentsViewController.m
+    - path: ../Riot/Modules/Common/Recents/Model
+    - path: ../Riot/Modules/Common/Recents/Views
+      excludes:
+        - "RecentTableViewCell.h"
+        - "RecentTableViewCell.m"
+        - "RecentTableViewCell.xib"
     - path: ../Riot/Modules/Common/SectionHeaders
     - path: ../Riot/Modules/Common/SegmentedViewController/SegmentedViewController.h
     - path: ../Riot/Modules/Common/SegmentedViewController/SegmentedViewController.m
     - path: ../Riot/Modules/Common/Views
     - path: ../Riot/Modules/Common/WebViewController
+    - path: ../Riot/Modules/CreateRoom
```

Comment:
Why do you add this resource?

Reply:
This resource is not needed for this PR. I removed it in last commit.